bin/vsm.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
```

Comment:
The file is still on a bin subdirectory, it should be on the root. Also, we should drop the .py for the entry point / main program.